### DISCUSSION TITLE
wz_mini_hacks on the new OG cams?

### DISCUSSION THREAD

**Discussion by tkwern** [gtxaspec/wz_mini_hacks#464] [Q&A]
Are there any thoughts/plans for wz_mini_hacks on the new OG cams?

**Comment by gtxaspec** [comment]
The new cameras use Realtek's Ameba chipsets which use a custom RTOS...

**Reply by mach7** [reply]
Does that mean that it wont be possible to put wz_mini_hacks on it?

**Comment by rickgitdone** [comment]
any more hardware details known on these?
